tendermint/src/proposal.rs

```diff
@@ -17,7 +17,7 @@ use crate::{Error, Kind};
 use bytes::BufMut;
 use std::convert::{TryFrom, TryInto};
 use tendermint_proto::types::Proposal as RawProposal;
-use tendermint_proto::{DomainType, Error as DomainTypeError};
+use tendermint_proto::{Error as DomainTypeError, Protobuf};
```

Comment:
```suggestion
use tendermint_proto::{Error as ProtobufError, Protobuf};
```
There are not many examples of this, but I guess it's worth renaming these too, for consistency.

Reply:
Done 👍